### ISSUE TITLE
Validate a11y conformance 

### ISSUE THREAD

**Issue by berdandy** [accessibilitywars/aw2-builds#9]
Theme CSS theoretically has some support for screen reader markup, but it has not been tested. Some visual elements may need to be hidden from voice, or some voice elements may need to be hidden from visuals.

- [ ] Check native windows Narrator
- [ ] Check native mac VoiceOver
- [ ] Check iOS VoiceOver
- [ ] Check Android TalkBack

Bonus points:
- [ ] Check linux speakup 😄 

See also https://www.a11yproject.com/checklist/

**Comment by MuwuM** [comment]
Important: All the [armory-embeds](https://github.com/madou/armory-embeds) do not provide any support for screen-reader at all. 

From my understanding they probably are ignored by most screenreaders, which means missing a lot of information on the build pages. We probably would need to PR or fork https://github.com/madou/armory-embeds to add support for it.

**Comment by MuwuM** [comment]
I just talked to @gw2princeps from [discretize](https://github.com/discretize). As [armory-embeds](https://github.com/madou/armory-embeds) looks liked it is not maintained anymore, they build their own solution [gw2-ui](https://github.com/discretize/discretize-ui/tree/main/gw2-ui) I'll have a look into it and check wether we could use it instead of amory-embeds. 

If yes, we could add my coding regarding key-bindings & some better support for screenreaders & keyboard-inputs to it and it would benefit both communities.
